[2025-03-06T00:58:47.508+0000] {processor.py:186} INFO - Started process (PID=3642) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T00:58:47.509+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T00:58:47.511+0000] {logging_mixin.py:190} INFO - [2025-03-06T00:58:47.511+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T00:58:47.517+0000] {logging_mixin.py:190} INFO - [2025-03-06T00:58:47.516+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T00:58:47.517+0000] {logging_mixin.py:190} INFO - [2025-03-06T00:58:47.517+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T00:58:47.517+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T00:58:47.531+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2025-03-06T00:59:45.453+0000] {processor.py:186} INFO - Started process (PID=4005) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T00:59:45.455+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T00:59:45.458+0000] {logging_mixin.py:190} INFO - [2025-03-06T00:59:45.457+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T00:59:45.464+0000] {logging_mixin.py:190} INFO - [2025-03-06T00:59:45.463+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T00:59:45.464+0000] {logging_mixin.py:190} INFO - [2025-03-06T00:59:45.464+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T00:59:45.464+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T00:59:45.477+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2025-03-06T01:00:45.655+0000] {processor.py:186} INFO - Started process (PID=4665) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:00:45.657+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:00:45.659+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:00:45.658+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:00:45.663+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:00:45.662+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:00:45.663+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:00:45.663+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:00:45.663+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:00:45.677+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2025-03-06T01:01:50.163+0000] {processor.py:186} INFO - Started process (PID=4910) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:01:50.165+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:01:50.167+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:01:50.166+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:01:50.170+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:01:50.170+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:01:50.171+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:01:50.171+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:01:50.171+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:01:50.184+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2025-03-06T01:02:50.109+0000] {processor.py:186} INFO - Started process (PID=5209) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:02:50.112+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:02:50.114+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:02:50.114+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:02:50.119+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:02:50.118+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:02:50.120+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:02:50.120+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:02:50.120+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:02:50.138+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2025-03-06T01:03:47.800+0000] {processor.py:186} INFO - Started process (PID=5388) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:03:47.802+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:03:47.804+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:03:47.804+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:03:47.808+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:03:47.808+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:03:47.809+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:03:47.809+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:03:47.809+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:03:47.823+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2025-03-06T01:04:53.409+0000] {processor.py:186} INFO - Started process (PID=5509) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:04:53.410+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:04:53.413+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:04:53.412+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:04:53.417+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:04:53.416+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:04:53.417+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:04:53.417+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:04:53.417+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:04:53.432+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2025-03-06T01:05:58.119+0000] {processor.py:186} INFO - Started process (PID=5633) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:05:58.121+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:05:58.127+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:05:58.126+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:05:58.131+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:05:58.130+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:05:58.132+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:05:58.132+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:05:58.132+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:05:58.151+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2025-03-06T01:07:03.173+0000] {processor.py:186} INFO - Started process (PID=6056) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:07:03.175+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:07:03.177+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:07:03.177+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:07:03.181+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:07:03.181+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:07:03.182+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:07:03.182+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:07:03.182+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:07:03.198+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2025-03-06T01:08:04.773+0000] {processor.py:186} INFO - Started process (PID=6355) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:08:04.775+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:08:04.778+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:08:04.777+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:08:04.782+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:08:04.781+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:08:04.782+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:08:04.782+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:08:04.782+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:08:04.798+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2025-03-06T01:09:04.819+0000] {processor.py:186} INFO - Started process (PID=6714) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:09:04.821+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:09:04.831+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:09:04.826+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:09:04.840+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:09:04.839+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:09:04.840+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:09:04.840+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:09:04.840+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:09:04.857+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2025-03-06T01:10:07.821+0000] {processor.py:186} INFO - Started process (PID=6838) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:10:07.823+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:10:07.825+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:10:07.824+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:10:07.831+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:10:07.830+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:10:07.832+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:10:07.831+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:10:07.832+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:10:07.852+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2025-03-06T01:11:09.157+0000] {processor.py:186} INFO - Started process (PID=6957) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:11:09.158+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:11:09.161+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:11:09.160+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:11:09.165+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:11:09.165+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:11:09.166+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:11:09.166+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:11:09.166+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:11:09.181+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2025-03-06T01:12:14.383+0000] {processor.py:186} INFO - Started process (PID=7077) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:12:14.385+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:12:14.387+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:12:14.387+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:12:14.391+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:12:14.390+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:12:14.391+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:12:14.391+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:12:14.391+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:12:14.404+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2025-03-06T01:13:16.986+0000] {processor.py:186} INFO - Started process (PID=7257) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:13:16.991+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:13:16.995+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:13:16.994+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:13:17.001+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:13:16.999+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:13:17.002+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:13:17.001+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:13:17.002+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:13:17.023+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2025-03-06T01:14:20.226+0000] {processor.py:186} INFO - Started process (PID=7439) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:14:20.234+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:14:20.240+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:14:20.240+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:14:20.253+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:14:20.250+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:14:20.254+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:14:20.254+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:14:20.254+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:14:20.292+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.076 seconds
[2025-03-06T01:15:19.472+0000] {processor.py:186} INFO - Started process (PID=7681) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:15:19.474+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:15:19.477+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:15:19.476+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:15:19.481+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:15:19.481+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:15:19.482+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:15:19.482+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:15:19.482+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:15:19.498+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2025-03-06T01:16:21.998+0000] {processor.py:186} INFO - Started process (PID=7861) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:16:22.007+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:16:22.015+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:16:22.013+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:16:22.023+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:16:22.022+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:16:22.023+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:16:22.023+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:16:22.023+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:16:22.050+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2025-03-06T01:17:23.828+0000] {processor.py:186} INFO - Started process (PID=8042) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:17:23.830+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:17:23.832+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:17:23.831+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:17:23.836+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:17:23.835+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:17:23.836+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:17:23.836+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:17:23.836+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:17:23.850+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2025-03-06T01:18:21.863+0000] {processor.py:186} INFO - Started process (PID=8382) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:18:21.864+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:18:21.866+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:18:21.866+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:18:21.869+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:18:21.869+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:18:21.870+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:18:21.870+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:18:21.870+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:18:21.882+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2025-03-06T01:19:30.187+0000] {processor.py:186} INFO - Started process (PID=8563) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:19:30.191+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-03-06T01:19:30.193+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:19:30.193+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:19:30.198+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:19:30.197+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-03-06T01:19:30.199+0000] {logging_mixin.py:190} INFO - [2025-03-06T01:19:30.199+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-03-06T01:19:30.199+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-03-06T01:19:30.224+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.056 seconds
